.circleci/config.yml

```diff
@@ -153,6 +154,15 @@ jobs:
                 min_instances: 1
                 max_instances: 2
             EOF
+      - run:
+          name: Create/Update AWS User policy
+          command: eval $(python ./manager/cluster_config_env.py cluster.yaml) && python ./dev/create_user.py $CORTEX_CLUSTER_NAME 388176669058 $CORTEX_REGION > $BASH_ENV
+      - run:
+          name: Wait for new keys to propagate in AWS
+          command: sleep 10
+      - run:
+          name: Verify configuration of credentials
+          command: aws sts get-caller-identity
```

Comment:
should we assert the user name is what we expect, or is it enough to check that this command succeeds?